cmd/zar/find/command.go

```diff
@@ -13,13 +13,22 @@ import (
 
 var Find = &charm.Spec{
 	Name:  "find",
-	Usage: "find [-d dir] <ip>",
+	Usage: "find [-d <dir>] <pattern>",
 	Short: "look through zar index files and displays matches",
 	Long: `
-"zar find" descends the directory given by the argument looking for bzng files that have
-a corresponding zar index and if that index contains the <ip> argument,
-then the path of the zng file is printed.
-The current version supports only IP address, but this will soon change.
+"zar find" descends the directory given by the -d option looking for bzng files
+that have a corresponding zar index conforming to the indicated <pattern>.
+The <pattern> argument has the form "field=value" (for field searches)
+or ":type=value" (for type searches).  For example, if type "ip" has been
+indexed then the IP 10.0.1.2 can be searched by saying
+
+	zar find -d /path/to/logs :ip=10.0.1.2
```

Comment:
Probably not for this PR, but I think for user-friendliness we should try to have the zar find expression syntax here be contained within the zql expression syntax... which would mean either adding support for `:type=value` to zql, or changing zar find to require `*=10.0.1.2`.

Reply:
I think this is a different discussion from search vs filter.  We're deliberately exposing index search as a foundational aspect of our architecture.  Coming up with the right index search syntax is still TBD.  This command-line syntax is a bit of a placeholder that maps directly onto the present indexing model.  I agree with your philosophy that it should be zql or maybe a subset of zql.

Reply:
Sounds good